src/callbacks.cc

```diff
@@ -31878,10 +31878,15 @@ void on_menu_item_check_updates_activate(GtkMenuItem*, gpointer) {
 }
 
 void on_menu_item_about_activate(GtkMenuItem*, gpointer) {
-	const gchar *authors[] = {"Hanna Knutsson", NULL};
+	const gchar *authors[] = {"Hanna Knutsson <hanna.knutsson@protonmail.com>, 2003-2007, 2016-2021",
+		"Sven Herzberg <herzi@gnome-de.org>, 2003",
+		"Jonathan Stickel <jjstickel@vcn.com>, 2009", NULL};
 	GtkWidget *dialog = gtk_about_dialog_new();
 	if(always_on_top) gtk_window_set_keep_above(GTK_WINDOW(dialog), always_on_top);
 	gtk_about_dialog_set_authors(GTK_ABOUT_DIALOG(dialog), authors);
+	if(_("translator-credits") != "translator-credits") {
+		gtk_about_dialog_set_translator_credits(GTK_ABOUT_DIALOG(dialog), _("translator-credits"));
+	}
 	gtk_about_dialog_set_comments(GTK_ABOUT_DIALOG(dialog), _("Powerful and easy to use calculator"));
 	gtk_about_dialog_set_license_type(GTK_ABOUT_DIALOG(dialog), GTK_LICENSE_GPL_2_0);
 	gtk_about_dialog_set_copyright(GTK_ABOUT_DIALOG(dialog), "Copyright © 2003–2007, 2008, 2016-2021 Hanna Knutsson");
```

Comment:
Discrepancy: "2003–2007" correctly uses an en-dash as a [range dash](https://en.wikipedia.org/wiki/Dash#En_dash), "2016-2021" uses a hyphen.

Reply:
The suggested code results in a compiler warning.
I will instead simply remove the string comparison ("GtkAboutDialog will detect if “translator-credits” is untranslated and hide the tab").

I will fix the dash/hyphen discrepancy.